e3nn/tensor_product.py

```diff
@@ -73,39 +73,70 @@ def forward(self, features_1, features_2):
 
 
 class LearnableTensorProduct(torch.nn.Module):
-    def __init__(self, Rs_mid_1, Rs_mid_2, mul_mid, Rs_out, selection_rule=o3.selection_rule):
+    def __init__(self, Rs_mid_1, Rs_mid_2, Rs_out,
+                 selection_rule=o3.selection_rule, groups=1):
```

Comment:
Also I propose that the order reflects the input to output order therefore I would put `selection_rule` before `Rs_out`